testing/qt5-qtserialport/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com> 
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=qt5-qtserialport
+pkgver=5.6.0
+pkgrel=1
+pkgdesc="Interface to serial ports for Qt application"
+url="http://qt-project.org/wiki/QtSerialPort"
+arch="all"
+license="LGPL"
+depends="qt5-qtbase"
+depends_dev="$depends"
+makedepends="$depends_dev qt5-qtbase-dev"
+install=""
+subpackages="$pkgname-dev"
+
+_ver=${pkgver/_/-}
+_ver=${_ver/beta0/beta}
+_ver=${_ver/rc0/rc}
+_V=${_ver/rc/RC}
```

Comment:
I took for a base APKBUILD file old "unmaintained/qtserialport/APKBUILD" so it is from there. Just fix that.

Is that ok that I'm deleting "unmaintained/qtserialport/APKBUILD" in this PR?